portfolio/src/main/webapp/maps.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Can be reached from index.html -->
+    <meta charset="UTF-8">
+    <title>Go-to Places</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAgtI2nxHdfJhgpJiy8Z0oJh7ZUsfS_ufc"></script>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body onload="createMap();">
+    <div id="first-back">
+      <h1>Here are some of my fav go-to places in Taipei!</h1>
+      <div id="map"></div>
+      <p>I would love to give you a tour when you come to Taipei!</p>
+    </div>
+  </body>
+</html>
```

Comment:
Very super nit: add an empty line below so we don't get distracted by the mark in code review.